aws-provider/postgresdb/README.md

```diff
@@ -0,0 +1,346 @@
+# `crossplane` PostgresDB example
+
+In this more complex example, we're going to walk through the process of
+enabling provider-families with `sts:AssumeRoleWithWebIdentity` to the
+management cluster, then using that to bootstrap a database using a secondary
+role assumed by crossplane for the actual build.
```

Comment:
When you mention "more complex example" you compare with...?  maybe a link to the simple one (just in case someone arrives here directly)

Reply:
and maybe add why we suggest this approach (segment role permissions...)

Reply:
Maybe I need to give some additional thought to the wording in this section. All of the AWS examples so far are "simple" in that none of them create more than a single piece of infrastructure and I made the assumption that because the repo is `crossplane-examples` users would look at this as a whole.

Let me consider how to rephrase this and update :small_airplane: